src/glim_ros/glim_ros.cpp

```diff
@@ -172,12 +175,16 @@ GlimROS::GlimROS(const rclcpp::NodeOptions& options) : Node("glim_ros", options)
   const std::string image_topic = config_ros.param<std::string>("glim_ros", "image_topic", "");
 
   // Subscribers
-  auto imu_qos = rclcpp::SensorDataQoS();
-  imu_qos.get_rmw_qos_profile().depth = 1000;
-  imu_sub = this->create_subscription<sensor_msgs::msg::Imu>(imu_topic, imu_qos, std::bind(&GlimROS::imu_callback, this, _1));
-  points_sub = this->create_subscription<sensor_msgs::msg::PointCloud2>(points_topic, rclcpp::SensorDataQoS(), std::bind(&GlimROS::points_callback, this, _1));
+  rclcpp::QoS qos{0};
+
+  get_qos(logger, config_ros, "imu_qos", qos);
```

Comment:
I would like to keep the default behavior unchanged.
Can we add a default QoS option to `get_qos()` that will be used when the associated configuration is not provided, and give the original QoS setting to it?